interfaces/admin_cli.py

```diff
@@ -0,0 +1,453 @@
+from cmd import Cmd
+import sys
+from datetime import datetime
+
+from controller.controller_error import ControllerError
+from interfaces.cli import EmsShell
+
+from models.admin import Admin
+from models.user import User, require_role
+from models.volunteer import Volunteer
+from models.plan import Plan
+import controller.plan_controller as plan_controller
+import controller.refugee_controller as refugee_controller
+import controller.volunteer_controller as volunteer_controller
+
+
+class AdminShell(EmsShell):
+    """
+    AdminShell is the command line interface for admin role.
+    """
+    admin_options = {"logout": 0, "profile": 1, "manage_plan": 2,
+                     "manage_volunteer_account": 3,
+                     "manage_refugee_profile": 4, "exit": "x"}
+
+    def admin_menu(self) -> None:
+        print("\033[100m\033[4m\033[1m{}\033[0m ".format("Admin Menu"))
+        print("\033[1mWelcome to EMS. Please choose an option to continue:\033[0m\n"
+              "[ 1 ] View my details\n"
+              "[ 2 ] Manage an emergency plan (Create/Close/View)\n"
+              "[ 3 ] Manage volunteer accounts and profile (Create/Edit/Deactivate/Delete)\n"
+              "[ 4 ] Manage Refugee Profile\n\n"
+              "[ 0 ] Log-out\n"
+              "[ X ] Exit\n")
+
+    def precmd(self, option: str) -> str:
+        """
+        Transfer option numbers to function name for Admin Menu
+        """
+        if option.isdigit() and int(option) in list(self.admin_options.values()):
+            return list(self.admin_options.keys())[list(self.admin_options.values()).index(int(option))]
+        elif option.upper() == 'X':
+            return list(self.admin_options.keys())[list(self.admin_options.values()).index(option.lower())]
+        else:
+            return "invalid_input"
+
+    def preloop(self) -> None:
+        """
+        Print out admin_menu before entering the shell.
+        :return:
```

Comment:
Could you remove the ::return:: please?

Reply:
removed!

Reply:
I am not sure but I can still see the :return:?